spec/system/signup_spec.rb

```diff
@@ -273,6 +273,7 @@
 
     if page.has_css?(".d-modal.login-modal", wait: 0)
       if page.has_css?("html.mobile-view", wait: 0)
+        expect(page).to have_css(".d-modal.is-animating")
```

Comment:
Hmm, what if stuff in the browser runs so quickly that `.is-animating` never gets seen by selenium?

Reply:
Yes that's a possible case, in my experience things don't run too fast though, but Im ok to not merge this. The spec might need more changes to be good anyways.